Cargo.toml

```diff
@@ -14,7 +14,7 @@ build = "src/build.rs"
 chrono = ">= 0.2"
 log = ">= 0.3"
 mime = ">= 0.1"
-url = ">= 0.5"
+url = "= 0.5"
```

Comment:
this probably ends up as merge conflict (and I'll revert it), but otherwise it doesn't build